lms/djangoapps/grades/tasks.py

```diff
@@ -23,7 +23,9 @@
 
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible, **kwargs):
+def recalculate_subsection_grade(
```

Comment:
In which code did you plan to fire the analytics events?
If the events are being fired from the celery workers, you'll need to pass the request-specific eventing-IDs to the celery task as well, right?